js/bytetrade.js

```diff
@@ -0,0 +1,1000 @@
+                        'orders/trades',  // Get detail match results
+                        'depositaddress', // Get deposit address
+                        'withdrawals',    // Get withdrawals info
+                        'deposits',       // Get deposit info
+                        'transfers',      // Get transfer info
+                        'transaction',    // Post create order\cancel order\withdraw\transfer transaction to blockchain
+                    ],
+                    'post': [
+                    ],
+                },
+                'private': {
+                    'get': [
+                    ],
+                    'post': [
+                    ],
+                },
+            },
+            'fees': {
+                'trading': {
+                    'taker': 0.0008,
+                    'maker': 0.0008,
+                },
+            },
+            'commonCurrencies': {
+                'CMT@18': 'CMT@18',
+            },
+            // exchange-specific options
+            'options': {
+            },
+            'exceptions': {
+                'vertify error': AuthenticationError,           // private key signature is incorrect
+                'transaction already in network': BadRequest,   // same transaction submited
+                'invalid argument': ArgumentsRequired,
+            },
+        });
+    }
+
+    async fetchCurrencies (params = {}) {
+        const currencies = await this.publicGetCurrencies (params);
+        const result = {};
+        for (let i = 0; i < currencies.length; i++) {
+            const currency = currencies[i];
+            const id = this.safeString (currency, 'code');
+            const name = this.safeValue (currency, 'name');
+            const nameId = name + '@' + id;
+            if (!(nameId in this.commonCurrencies)) {
+                this.commonCurrencies[nameId] = name;
+            }
+            const code = this.commonCurrencyCode (nameId);
+            const active = currency['active'];
+            // const name = this.safeString (currency, 'display-name');
+            const limits = this.safeValue (currency, 'limits');
+            const deposit = this.safeValue (limits, 'deposit');
+            let maxDeposit = this.safeFloat (deposit, 'max');
+            if (maxDeposit === -1) {
+                maxDeposit = undefined;
+            }
+            const withdraw = this.safeValue (limits, 'withdraw');
+            let maxWithdraw = this.safeFloat (withdraw, 'max');
+            if (maxWithdraw === -1) {
+                maxWithdraw = undefined;
+            }
+            result[code] = {
+                'id': id,
+                'code': code,
+                'type': 'crypto',
+                'name': name,
+                'active': active,
+                'fee': undefined,
+                'basePrecision': this.safeValue (currency, 'basePrecision'),
+                'transferPrecision': this.safeValue (currency, 'transferPrecision'),
```

Comment:
these are not unified fields, please read: https://github.com/ccxt/ccxt/wiki/Manual#market-structure

Reply:
... or https://github.com/ccxt/ccxt/wiki/Requirements )

Reply:
thank you, i have fixed it.